spec/PeriodicBackgroundSync-index.bs

```diff
@@ -71,57 +71,74 @@ Introduction {#intro}
+Note: The actual interval at which [=periodicsync event|periodicsync events=] are fired MUST be greater than or equal to this.
 
-A <dfn>time to fire</dfn> (a timestamp), which is the soonest time [=calculate a time to fire| calculated=] by the user agent at which it is appropriate to fire the next [=periodicSync event=] for the [=periodicsync registration=].
+A <dfn>time to fire</dfn> (a timestamp), which is the soonest time [=calculate a time to fire| calculated=] by the user agent at which it is appropriate to fire the next [=periodicsync event=] for the [=periodic sync registration=].
 
-A <dfn>count of retries</dfn> (a number), which is the number of retries attempted for the most recent [=periodicSync event=] for the [=periodicsync registration=].
+A <dfn>count of retries</dfn> (a number), which is the number of retries attempted for the most recent [=periodicsync event=] for the [=periodic sync registration=]. This is initially set to zero.
 
-A <dfn>registration state</dfn>, which is one of <dfn>pending</dfn>, <dfn>firing</dfn>, or <dfn>reregisteredWhileFiring</dfn>. It is initially set to [=pending=].
+A <dfn>state</dfn>, which is one of "<code>pending</code>", "<code>firing</code>", "<code>suspended</code>" or "<code>reregistered-while-firing</code>". It is initially set to "<code>pending</code>".
 </div>
 
+## <dfn>Periodic Sync Scheduler</dfn> ## {#periodic-sync-scheduler-construct}
+The [=periodic sync scheduler=] is responsible for scheduling firing of [=periodicsync events=].
+In response to these triggers, the scheduler either schedules delayed processing to fire a [=periodicsync event=] at the appropriate time in the future, or canceling such scheduling.
+
+  <div dfn-for="periodic sync scheduler">
+    The scheduler maintains:
+    * A <dfn>scheduler processing queue</dfn>, initially the result of [=starting a new parallel queue=].
+    * <dfn>Scheduled registrations</dfn>, a [=list=] of [=periodic sync registration| periodic sync registrations=], initially an empty [=list=].
+    * The <dfn>time till the next scheduled periodicsync event</dfn>, a [=map=], where each key is an [=origin=], and each item is a long long representing time delay. The keys of this map are the [=origin|origins=] associated with the [=periodic sync registration/service worker registration|service worker registrations=] of [=scheduled registrations=].
+  </div>
+
+  The scheduler [=process periodic sync registrations|processes periodic sync registrations=] continuously on the [=periodic sync scheduler/scheduler processing queue=].
+  Note: Browsers may suspend this processing loop when there are no [=active periodic sync registrations=] to conserve resources.
```

Comment:
If you put `Note: …` on its own line, bikeshed will mark it up as a non-normative note. See the background fetch spec for examples.